src/main/java/com/arcanewarrior/disguises/Disguise.java

```diff
@@ -0,0 +1,24 @@
+package com.arcanewarrior.disguises;
+
+import net.minestom.server.entity.EntityType;
+import net.minestom.server.entity.LivingEntity;
+import net.minestom.server.entity.Player;
+import org.jetbrains.annotations.NotNull;
+
+public final class Disguise extends LivingEntity {
+
+    private final Player player;
+
+    public Disguise(@NotNull EntityType entityType, @NotNull Player player) {
+        super(entityType);
+        this.player = player;
+    }
+
+    public Player getPlayer() {
+        return player;
+    }
+
+    public void unDisguise() {
+        MinestomDisguises.getInstance().getDisguiseManager().unDisguisePlayer(getPlayer());
```

Comment:
Since disguises are constructed by the disguise manager, I'd recommend you pass in the DisguiseManager to the constructor instead of static access.
Also, please don't have undisguise capitalized like that.